core/chain_makers_test.go

```diff
@@ -118,7 +118,7 @@ func TestGenerateWithdrawalChain(t *testing.T) {
 	// enforce that withdrawal indexes are monotonically increasing from 0
 	var (
 		withdrawalIndex uint64
-		head            = blockchain.CurrentBlock().NumberU64()
+		head            = blockchain.CurrentBlock().Number.Uint64()
```

Comment:
This change was already included in https://github.com/ethereum/go-ethereum/pull/26802